dbos/_sys_db.py

```diff
@@ -1852,6 +1852,65 @@ def check_connection(self) -> None:
             dbos_logger.error(f"Error connecting to the DBOS system database: {e}")
             raise
 
+    def garbage_collect(
+        self, time_threshold_ms: Optional[int], rows_threshold: Optional[int]
```

Comment:
Same question for `time_threshold_ms` here... it's converted to seconds anyways.